CHANGELOG.md

```diff
@@ -8,11 +8,12 @@ nav_order: 1
 
 ## [4.0.0] - YYYY-MM-DD
 
-- Mark `ip_filter` and `namespaces` deprecated
+- Mark `ip_filter` and `namespaces` user configuration options as deprecated
 - Make schema fields use strict types instead of string
 - Add support for strict types in diff functions
 - Add `stateupgrader` package which aims to assist with upgrading from v3.x to v4.0.0
 - Remove deprecated resources (with data sources): `aiven_database`, `aiven_service_user`, `aiven_vpc_peering_connection`
+- Remove deprecated resources: `aiven_flink_table` and `aiven_flink_job`
```

Comment:
Do we still need to have [flink resource](https://registry.terraform.io/providers/aiven/aiven/latest/docs/resources/flink) at all?
They might change user config too.

Reply:
yes Flink service will stay but some user configuration options will be gone